### PR TITLE
Additional dynamic init thresholds

### PR DESCRIPTION
**Pull request by CanCanZeng**
add init_dyn_min_movement flag to make dynamic intialization more robust

### CODE REVIEW

File: ov_init/src/dynamic/DynamicInitializer.cpp [hunk 1]
```diff
@@ -909,6 +909,15 @@ bool DynamicInitializer::initialize(double &timestamp, Eigen::MatrixXd &covarian
     return state_imu;
   };
 
+  // Calculate the distance between the head and the tail.
+  // If the movement is smaller than threshold, the parallax is not enough and initialization is not allowed
+  Eigen::Vector3d movement = get_pose(newest_cam_time).segment<3>(4) - get_pose(oldest_camera_time).segment<3>(4);
```
Comment (goldbattle) [inline ov_init/src/dynamic/DynamicInitializer.cpp:914]: Please use `.block(4, 0, 3, 1)` here.

**Comment by goldbattle** [comment]
Should also likely check if the velocity is near zero too as we should likely use static in this case.

**Comment by CanCanZeng** [comment]
I add a flag to control the minimum velocity for dynamic initialization, and I suggest to use two parameters to control the window length for static and dynamic initialization respectively

**Comment by goldbattle** [comment]
I have thought about this, and will close this feature for now. Maybe will revisit in the future if the dynamic initialization module is cleaned up.

**Pull request closed without merging**